reference/physics.md

```diff
@@ -4,7 +4,7 @@
 Physics {
   SFFloat density       1000  # (kg/m^3) -1 or > 0
   SFFloat mass          -1    # (kg) -1 or > 0
-  SFVec3f centerOfMass  0 0 0 # (-inf,inf)
+  MFVec3f centerOfMass  0 0 0 # (-inf,inf)
```

Comment:
the default value should be `[ ]` and I'm not sure that the comment still makes sense